package.json

```diff
@@ -62,7 +62,7 @@
     }
   },
   "lint-staged": {
-    "**/*.+(ts|tsx|js|jsx|json|sol|java|yml|yaml)": "prettier --write"
+    "**/*.+(ts|tsx|js|jsx|json|sol|java|yml|yaml|md)": "prettier --write"
```

Comment:
i had a [lint error](https://github.com/valora-inc/wallet/actions/runs/10807252211/job/29977661197?pr=5969) on the readme .md file, so added this